Makefile.amigaos

```diff
@@ -7,43 +7,54 @@
 ##########################################################################
 
 ##########################################################################
-# Default: Build iGame
+# Default: Build iGame with standard optimizations and 000 support
 ##########################################################################
 all: iGame
 
+##########################################################################
+# Set up version and date properties
+##########################################################################
+
+DATE = $(shell date --iso=date)
```

Comment:
This worked for AmigaOS 4.1 with the SDK installed, but I believe that's because the GNU date utility took priority in the path in that instance. You're right that it probably won't work normally, especially as AmigaOS has its own date utility.